src/learning-catalogue/service/termsAndConditionsService.ts

```diff
@@ -0,0 +1,33 @@
+import {RestService} from './restService'
+import {TermsAndConditionsFactory} from '../model/factory/termsAndConditionsFactory'
+import {TermsAndConditions} from '../model/termsAndConditions'
+
+export class TermsAndConditionsService {
```

Comment:
These services could all be generic - create an instance passing a factory and a resource path. Need to consider variable replacement in the paths but there's a lot of duplication going on that could be simplified (maybe?).